drivers/i2c/bme280_driver.go

```diff
@@ -113,6 +114,7 @@ func (d *BME280Driver) initHumidity() (err error) {
 	d.hc.h5 = 0 + (int16(addrE6) << 4) | (int16(addrE5) >> 4)
 
 	d.connection.WriteByteData(bme280RegisterControlHumidity, 0x3F)
+	d.connection.WriteByteData(bmp280RegisterControl, 0x3F)
```

Comment:
Hi @ssorc I'd really like to merge this in preparation for the next Gobot release. Any chance of getting that update soon? :)

Thank you!